pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+**Description:**
+
+Describe in a couple of sentences what this PR adds
+
+**Related Issue:**
+
+Link to the related Issue
+
+**Configuration instructions:**
+
+List any non-trivial configuration instructions (if any)
+
+**Testing:**
+
+Explain what kind of testing was performed and list any
+testing instructions (if needed)
```

Comment:
Does this need to be a separate line?

Reply:
Yep. For some reason this file was added in without me noticing it. Had planned to add it to another PR, but yeah.
Added in the rest of the file.